packages/sirius-web/frontend/sirius-web-application/src/extension/DefaultExtensionRegistry.tsx

```diff
@@ -91,6 +91,10 @@ import { ProjectSettingTabContribution } from '../views/project-settings/Project
 import { projectSettingsTabExtensionPoint } from '../views/project-settings/ProjectSettingsViewExtensionPoints';
 import { ellipseNodeStyleDocumentTransform } from './EllipseNodeDocumentTransform';
 import { referenceWidgetDocumentTransform } from './ReferenceWidgetDocumentTransform';
+import { OmniboxCommandOverrideContribution } from '@eclipse-sirius/sirius-components-omnibox';
```

Comment:
These changes should be made in the papaya frontend package not the Sirius Web one